internal/store/deployment_test.go

```diff
@@ -33,7 +33,7 @@ var (
 	depl2Replicas int32 = 5
 
 	depl1MaxUnavailable = intstr.FromInt(10)
-	depl2MaxUnavailable = intstr.FromString("20%")
+	depl2MaxUnavailable = intstr.FromString("25%")
```

Comment:
With rounding up, this will fail to produce the existing expected value of 1. With my change (rounding down) we still produce the existing expected value.